src/compiler/factory/emitHelpers.ts

```diff
@@ -844,6 +879,66 @@ namespace ts {
             };`
     };
 
+    export const classPrivateMethodGetHelper: UnscopedEmitHelper = {
+        name: "typescript:classPrivateMethodGet",
+        importName: "__classPrivateMethodGet",
+        scoped: false,
+        text: `
+            var __classPrivateMethodGet = (this && this.__classPrivateMethodGet) || function (receiver, instances, fn) {
+                if (!instances.has(receiver)) {
+                    throw new TypeError("attempted to get private method on non-instance");
+                }
+                return fn;
+            };`
+    };
+
+    export const classPrivateReadonlyHelper: UnscopedEmitHelper = {

```

Comment:
Also, we error on incorrect usage in the checker. Is there a reason to also emit a runtime error?

Reply:
@rbuckton We do error in the checker for cases where the access can be statically checked. But accessing a private field on `any` is not an error, so runtime checks and errors are still a must.

Reply:
We could remove the runtime check, but it would mean that if we ever get a flag like `transpileOnly` the net result would be to emit code that might work when the equivalent standard code would not. Also I think you can skip checking i you use the compiler API. 

I think since we removed the runtime helpers and just kept the two, the amount of extra code to keep this is minimal.